syntax = "proto3";

message HBReqDTO {
  int32 offset = 1;       // Offset value
  int32 time = 2;         // Timestamp of the request
  int32 csq = 3;          // Carrier Signal Quality (CSQ)
  string dtu_serial_number = 4;  // DTU serial number
  string device_serial_number = 5; // Device serial number
}

message HBResDTO {
  int32 offset = 1;       // Offset value
  int32 time = 2;         // Timestamp of the response
  string time_ymd_hms = 3;     // Timestamp in the format YMD_HMS
}
